### ISSUE TITLE
Fileset closing is not waiting for the data in the run_stream_fileset to be complete

### ISSUE THREAD

**Issue by vytjan** [dmwm/T0#4474]
Noticed only now, but back in November we had a case where multiple filesets closed before all streamer files from respective streams were transferred to Tier0.
There is a WMCore GH issue addressing this:
https://github.com/dmwm/WMCore/issues/8952

I tried re-opening the Run326546_StreamHIExpress fileset and then the unused streamer files were added to the fileset properly.

**Comment by amaltaro** [comment]
Dirk, this is the DAO querying for filesets that can be closed:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMBS/MySQL/Fileset/ListClosable.py

trying to make sense of this query in plain english:
* workflow entry needs to be injected or have that alt_fs_close=1 (L22)
* have 0 running jobs, i.e., no jobs in a state != cleanout (L35)
* have 0 files acquired - wmbs_sub_files_acquired
* have 0 files available - wmbs_sub_files_available
* and, of course, be open :)

Did I miss anything?
BTW, since the T0 agent doesn't run WorkQueue/WMBSHelper code, the only place responsible for setting a fileset to open or not is in the JobArchiverPoller component.

**Comment by hufnagel** [comment]
The original design was that the Tier0 controlled when a workflow was set to injected, we only did it when all the data from P5 was transferred and the bookkeeping consistent,

Guess I have to check that code. Could still be operational problems, i.e. P5 bookkeeping and transfers were changed after the fact.

**Comment by vytjan** [comment]
Those files were transferred some time later than the rest of streamer files from those two runs. 
Therefore, it could still be some operational problems at P5.

**Comment by germanfgv** [comment]
As we haven's seen this issue in years,  I'll close this issue
